tests/models/post_image_manager_test_norun.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
```

Comment:
To fix this class I think we can fix this by passing in the root directory as a parameter to the clear method and remove the reference to Rails.root